test/httpcrashtest.cpp

```diff
@@ -179,28 +177,20 @@ void HTTPCrashTest::testRecoverAfterKitCrash()
     const char* testname = "recoverAfterKitCrash ";
     try
     {
+        TST_LOG("Loading document");
         std::shared_ptr<http::WebSocketSession> socket1
             = loadDocAndGetSession(_socketPoll, "empty.odt", _uri, testname);
 
-        TST_LOG("Allowing time for kits to spawn and connect to wsd to get cleanly killed");
+        TST_LOG("Allowing time for kit to connect to wsd to get cleanly killed");
         std::this_thread::sleep_for(std::chrono::seconds(1));
 
         TST_LOG("Killing coolkit instances.");
+        killDocKitProcesses(testname);
+        waitForKitPidsReady(testname);
 
-        killLoKitProcesses();
-        countCoolKitProcesses(0);//, std::chrono::seconds(1));
-
-        // We expect the client connection to close.
         TST_LOG("Reconnect after kill.");
-
         std::shared_ptr<http::WebSocketSession> socket2 = loadDocAndGetSession(
             _socketPoll, "empty.odt", _uri, testname, /*isView=*/true, /*isAssert=*/false);
-        if (!socket2)
```

Comment:
seems reasonable; coolwsd should be up and running reliably in this mode.

Reply:
Yup, it was wasting 20s by trying to connect 1s before it was ready, then waiting to time out and retry.